### DISCUSSION TITLE
login to vercel.com is down

### DISCUSSION THREAD

**Discussion by richleach** [vercel/community#5608] [Account Issue]
### Question

Apparently I have several different logins with vercel.com and I can only reach and interact with the one, when in fact I need the other. I have clicked the Log Out link and tried logging back in to no avail. Your help links are useless and contact forms only go to sales??? Please advise, and thank you.

**Comment by richleach** [comment]
When I click the "Continue With Github" button it apparently uses my last GitHub account's credentials, when in fact I have several different GitHub accounts. I never get the chance to designate which GitHub account I'd like to use to login. What do you suggest?

**Comment by acarmagnani** [comment]
I've been having issues logging in as well. I've been trying to sign up but the error message "This user account is blocked." keeps appearing for me and I can't seem to find a solution for this problem on the internet.
